DEV-10/src/DoubleArrays.java

```diff
@@ -0,0 +1,99 @@
+import java.util.ArrayList;
+
+/**
+ * This class make array, that contains only non unique nembers from two-dimensional array.
+ */
+public class DoubleArrays {
+
+  /**
+   * Compare two double numbers with precision 1e-10.
+   * @param first - double number.
+   * @param second - double number.
+   * @return result of comparing.
+   */
+  private boolean compare(double first, double second) {
```

Comment:
I doubt it works correctly for the numbers less than your precision, that is sad